### ISSUE TITLE
Loss is larger when using a 2B-parameter model

### ISSUE THREAD

**Issue by YUHANG-Ma** [lucidrains/DALLE2-pytorch#213]
Hi everyone, I modified the model to 2B parameters with the following config.
![image](https://user-images.githubusercontent.com/72591225/180355755-424f94b2-e186-486a-a936-32a40e634d9d.png)
However, when I am training, in the case of the same epoch, the effect of the loss function convergence (to about 0.06) is not as good as when the model parameters are relatively small (900M) which is 0.04. I am wondering if this is a normal phenomenon because of more parameters of UNet.
